pkg/api/steve/proxy/proxy_test.go

```diff
@@ -41,54 +43,103 @@ func TestLocalCluster(t *testing.T) {
 			userCanAccessLocal:     false,
 			desiredResponseCode:    http.StatusForbidden,
 			desiredResponseMessage: "",
 		},
 		{
-			name:                   "core types without local cluster access",
+			name:                   "management allowed crd without local cluster access",
+			requestPath:            "/apis/management.cattle.io/v3/podsecurityadmissionconfigurationtemplates",
+			userCanAccessLocal:     false,
+			desiredResponseCode:    defaultResponseCode,
+			desiredResponseMessage: defaultResponseMessage,
+		},
+		{
+			name:                   "management allowed crd with local cluster access",
+			requestPath:            "/apis/management.cattle.io/v3/podsecurityadmissionconfigurationtemplates",
+			userCanAccessLocal:     true,
+			desiredResponseCode:    defaultResponseCode,
+			desiredResponseMessage: defaultResponseMessage,
+		},
+		{
+			name:                   "management allowed crd with individual resource without local cluster access",
+			requestPath:            "/apis/management.cattle.io/v3/podsecurityadmissionconfigurationtemplates/mytemplate",
+			userCanAccessLocal:     false,
+			desiredResponseCode:    defaultResponseCode,
+			desiredResponseMessage: defaultResponseMessage,
+		},
+		{
+			name:                   "management allowed crd with individual resource's status without local cluster access",
+			requestPath:            "/apis/management.cattle.io/v3/podsecurityadmissionconfigurationtemplates/mytemplate/status",
+			userCanAccessLocal:     false,
+			desiredResponseCode:    defaultResponseCode,
+			desiredResponseMessage: defaultResponseMessage,
+		},
+		{
+			name:                   "core type without local cluster access",
 			requestPath:            "/api/v1/pods",
 			userCanAccessLocal:     false,
 			desiredResponseCode:    defaultResponseCode,
 			desiredResponseMessage: defaultResponseMessage,
 		},
 		{
-			name:                   "core types with local cluster access",
+			name:                   "core type with local cluster access",
 			requestPath:            "/api/v1/pods",
 			userCanAccessLocal:     true,
 			desiredResponseCode:    defaultResponseCode,
 			desiredResponseMessage: defaultResponseMessage,
 		},
 		{
-			name:                   "non-mgmt types without local cluster access",
+			name:                   "non management type without local cluster access",
 			requestPath:            "/apis/provisioning.cattle.io/v1/clusters",
 			userCanAccessLocal:     false,
 			desiredResponseCode:    defaultResponseCode,
 			desiredResponseMessage: defaultResponseMessage,
 		},
 		{
-			name:                   "non-mgmt types with local cluster access",
+			name:                   "non management type with local cluster access",
 			requestPath:            "/apis/provisioning.cattle.io/v1/clusters",
 			userCanAccessLocal:     true,
 			desiredResponseCode:    defaultResponseCode,
 			desiredResponseMessage: defaultResponseMessage,
 		},
+		{
```

Comment:
Add a test case that has a different version  such as `management.cattle.io/v4/fake-crd`